DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/UploadDelegatesViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.ViewModels.Register.RegisterDelegateByCentre;
+    using Microsoft.AspNetCore.Http;
+
+    public class UploadDelegatesViewModel : WelcomeEmailViewModel

```

Comment:
I assume this means we get all the date validation?

Reply:
Yes